app/src/main/java/org/wikipedia/notifications/NotificationActivity.kt

```diff
@@ -169,7 +189,7 @@ class NotificationActivity : BaseActivity(), NotificationItemActionsDialog.Callb
     private val orContinueNotifications: Unit
         get() {
             binding.notificationsProgressBar.visibility = View.VISIBLE
-            disposables.add(ServiceFactory.get(WikiSite(Service.COMMONS_URL)).getAllNotifications("*", if (displayArchived) "read" else "!read", currentContinueStr)
+            disposables.add(ServiceFactory.get(WikiSite(Service.COMMONS_URL)).getAllNotifications(getDelimitedFilteredWikiList(), if (displayArchived) "read" else "!read", currentContinueStr)
```

Comment:
I have also updated this in the #2801, would it be possible to make the base branch to `notification-list-update` branch?